src/services/MyClassDetailsService.ts

```diff
@@ -10,7 +10,7 @@ export const getMyCohortMemberList = async ({
   filters,
 }: cohortMemberList): Promise<any> => {
   const apiUrl: string = `${process.env.NEXT_PUBLIC_BASE_URL}/cohortmember/list`;
-  filters = { ...filters, role: 'Student' };
+  filters = { ...filters, role: 'Student', status:["dropout" , "active"] };
```

Comment:
@Rushikesh-Sonawane99 for role and status make an enum and keep it in the constant file and use across the project

Reply:
@itsvick Added enum and used accordingly